src/browser/custom_context.py

```
@@ -16,4 +16,81 @@ def __init__(
         browser: "Browser",
         config: BrowserContextConfig = BrowserContextConfig()
     ):
-        super(CustomBrowserContext, self).__init__(browser=browser, config=config)
\ No newline at end of file
+
+        super(CustomBrowserContext, self).__init__(browser=browser, config=config)
```

Ruff (0.8.2)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)